### PR TITLE
Use dignified.js

### PR DESCRIPTION
**Pull request by dignifiedquire**
Ref https://github.com/dignifiedquire/dignified.js/issues/1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,5 +44,14 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/diasdavid/js-peer-id.git"
+  },
+  "dignified": {
+    "webpack": {
+      "resolve": {
+        "alias": {
+          "node-forge": "../vendor/forge.bundle.js"
```
Comment (diasdavid) [inline package.json]: nice! Was looking for a note on how to add specifics like this :)

Comment (dignifiedquire) [inline package.json]: This is currently only possible for `webpack`.  When the need arises we can add it for others as well

**Comment by diasdavid** [comment]
- [ ] add .npmignore

File: src/index.js [hunk 2]
```diff
@@ -9,10 +10,8 @@ const forge = require('node-forge')
 const protobuf = require('protocol-buffers')
 const path = require('path')
 
-const isNode = !global.window
-
 // protobuf read from file
-const messages = isNode ? protobuf(fs.readFileSync(path.resolve(__dirname, 'pb/crypto.proto'))) : protobuf(require('buffer!./pb/crypto.proto'))
+const messages = protobuf(fs.readFileSync(path.resolve(__dirname, '../protos/crypto.proto')))
```
Comment (xicombd) [inline src/index.js]: This seems to be braking the browser tests on `js-ipfs`. Why did you remove the `isNode` check?
cc @dignifiedquire @diasdavid

Comment (dignifiedquire) [inline src/index.js]: Because this way we can use `brfs` to always load the file. As soon as `js-ipfs` uses dignified.js it should work again, or you can manually add this config: https://github.com/dignifiedquire/dignified.js/blob/master/config/webpack.js#L89-L92

Comment (xicombd) [inline src/index.js]: Got it, thanks @dignifiedquire!

Comment (diasdavid) [inline src/index.js]: Did the tests not run for you @xicombd ?

Comment (xicombd) [inline src/index.js]: No, `js-ipfs`'s browser tests before https://github.com/ipfs/js-ipfs/pull/132 were not passing.

Comment (diasdavid) [inline src/index.js]: got it :)
